src/sas/qtgui/Perspectives/Fitting/FittingUtilities.py

```diff
@@ -109,44 +139,58 @@ def addParametersToModel(parameters, kernel_module, is2D):
                 item1_5.setEditable(False)
                 poly_item.appendRow([item1_1, item1_2, item1_3, item1_4, item1_5])
                 break
+
             # Add the polydisp item as a child
             item1.appendRow([poly_item])
+
         # Param values
         item2 = QtGui.QStandardItem(str(param.default))
-        # TODO: the error column.
-        # Either add a proxy model or a custom view delegate
-        #item_err = QtGui.QStandardItem()
         item3 = QtGui.QStandardItem(str(param.limits[0]))
         item4 = QtGui.QStandardItem(str(param.limits[1]))
-        item5 = QtGui.QStandardItem(param.units)
+        item5 = QtGui.QStandardItem(str(param.units))
```

Comment:
Yes, but we were playing with placing a list of option choices in the units field (probably won't go ahead but still) so this prevents any breakage, just in case